spec/models/rent_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Rent, type: :model do
```

Comment:
I think this file was generated when you run rails generate model. Let's remove it. Maybe we will add a new one later